ext/GDBM_File/t/count.t

```diff
@@ -0,0 +1,34 @@
+#!./perl -w
+use strict;
+
+use Test::More;
+use Config;
+
+BEGIN {
+    plan(skip_all => "GDBM_File was not built")
+	unless $Config{extensions} =~ /\bGDBM_File\b/;
+
+    # https://rt.perl.org/Public/Bug/Display.html?id=117967
+    plan(skip_all => "GDBM_File is flaky in $^O")
+        if $^O =~ /darwin/;
+
+    plan(tests => 3);
+    use_ok('GDBM_File');
+ }
+
+unlink <Op_dbmx*>;
+
+my %h;
+my $db = tie(%h, 'GDBM_File', 'Op_dbmx', GDBM_WRCREAT, 0640);
```

Comment:
This test should probably use a tempdir from `File::Temp`

Reply:
Yes, that's reasonable.  I did it that way because all existing tests do so.